server/src/test/kotlin/com/kamelia/hedera/rest/UserTest.kt

```diff
@@ -818,6 +859,192 @@ class UserTest {
+                ),
+                Arguments.of(
+                    Named.of("regular user", user),
+                    Named.of("admin", UUID.fromString("00000000-0000-0011-0003-000000000003")),
+                    Named.of("enabled", true),
+                    HttpStatusCode.Forbidden
+                ),
+                Arguments.of(
+                    Named.of("regular user", user),
+                    Named.of("admin", UUID.fromString("00000000-0000-0011-0003-000000000004")),
+                    Named.of("disabled", false),
+                    HttpStatusCode.Forbidden
+                ),
+                Arguments.of(
+                    Named.of("regular user", user),
+                    Named.of("regular", UUID.fromString("00000000-0000-0011-0003-000000000005")),
+                    Named.of("enabled", true),
+                    HttpStatusCode.Forbidden
+                ),
+                Arguments.of(
+                    Named.of("regular user", user),
+                    Named.of("regular", UUID.fromString("00000000-0000-0011-0003-000000000006")),
+                    Named.of("disabled", false),
+                    HttpStatusCode.Forbidden
+                ),
+                Arguments.of(
+                    Named.of("guest", guest),
+                    Named.of("owner", UUID.fromString("00000000-0000-0011-0004-000000000001")),
+                    Named.of("enabled", true),
+                    HttpStatusCode.Unauthorized
+                ),
+                Arguments.of(
+                    Named.of("guest", guest),
+                    Named.of("owner", UUID.fromString("00000000-0000-0011-0004-000000000002")),
+                    Named.of("disabled", false),
+                    HttpStatusCode.Unauthorized
+                ),
+                Arguments.of(
+                    Named.of("guest", guest),
+                    Named.of("admin", UUID.fromString("00000000-0000-0011-0004-000000000003")),
+                    Named.of("enabled", true),
+                    HttpStatusCode.Unauthorized
+                ),
+                Arguments.of(
+                    Named.of("guest", guest),
+                    Named.of("admin", UUID.fromString("00000000-0000-0011-0004-000000000004")),
+                    Named.of("disabled", false),
+                    HttpStatusCode.Unauthorized
+                ),
+                Arguments.of(
+                    Named.of("guest", guest),
+                    Named.of("regular", UUID.fromString("00000000-0000-0011-0004-000000000005")),
+                    Named.of("enabled", true),
+                    HttpStatusCode.Unauthorized
+                ),
+                Arguments.of(
+                    Named.of("guest", guest),
+                    Named.of("regular", UUID.fromString("00000000-0000-0011-0004-000000000006")),
+                    Named.of("disabled", false),
+                    HttpStatusCode.Unauthorized
+                ),
```

Comment:
Introduce new users in the dataset (as in new UUID), so that if these tests fail, they won't impact other ones